.dockerignore

```diff
@@ -35,3 +35,7 @@ token-tracker/
 # Legacy V1 contracts bindings.
 # We won't generate new bindings in the docker build process, but use the existing ones.
 !pkg/chain/gen
+
```

Comment:
Can we add a comment to clarify this section?

```suggestion

# Contracts addresses.
```

Reply:
[2e24cca](https://github.com/keep-network/keep-core/pull/3410/commits/2e24cca713dc3d508669376918c4bda602859bf0)